truss/tests/conftest.py

```diff
@@ -676,6 +676,18 @@ def custom_model_truss_dir_with_pre_and_post_description(tmp_path):
     yield dir_path
 
 
+@pytest.fixture
+def custom_model_truss_dir_with_hidden_files(tmp_path):
+    truss_dir_path: Path = tmp_path / "custom_model_truss_dir_with_hidden_files"
```

Comment:
we should add a few files we don't want to ignore. Like a model.py or something.